net/DescriptionViewController.m

```diff
@@ -90,12 +121,12 @@ -(void)viewDidLoad
 
 -(void)setDataToView
 {
-    self.titleLabel.text = self.currentIssue.name;
+    self.titleLabel.text = [CurrentItems sharedItems].issue.name;
```

Comment:
This approach is realised with currentUser and MapView Controller. I've left property currentUser, and update it from singleton.
In methods, which don't use singleton often-times, I use it directly.